examples/profile/main.go

```diff
@@ -48,6 +48,104 @@ func home(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func sourceConstraints(w http.ResponseWriter, req *http.Request) {
+	sdkID := os.Getenv("YOTI_CLIENT_SDK_ID")
+
+	key, err := ioutil.ReadFile(os.Getenv("YOTI_KEY_FILE_PATH"))
+	if err != nil {
+		errorPage(w, req.WithContext(context.WithValue(
+			req.Context(),
+			contextKey("yotiError"),
+			fmt.Sprintf("Unable to retrieve `YOTI_KEY_FILE_PATH`. Error: `%s`", err),
+		)))
+		log.Printf("Unable to retrieve `YOTI_KEY_FILE_PATH`. Error: `%s`", err)
+		return
+	}
+
+	client := yoti.Client{
+		SdkID: sdkID,
+		Key:   key,
+	}
+
+	constraint := (&yoti.SourceConstraintBuilder{}).New().WithDrivingLicence("").WithPassport("").Build()
+	scenario := (&yoti.DynamicScenarioBuilder{}).New().WithPolicy(
+		(&yoti.DynamicPolicyBuilder{}).New().WithFullName(constraint).WithStructuredPostalAddress(constraint).Build(),
+	).WithCallbackEndpoint("/profile").Build()
+
+	share, err := yoti.CreateShareURL(&client, &scenario)
+	if err != nil {
+		errorPage(w, req.WithContext(context.WithValue(
+			req.Context(),
+			contextKey("yotiError"),
+			fmt.Sprintf("%s", err),
+		)))
+		return
+	}
+
+	templateVars := map[string]interface{}{
+		"yotiClientSdkID": sdkID,
+		"yotiShareURL":    share.ShareURL,
+	}
+
+	t, err := template.ParseFiles("dynamic-share.html")
```

Comment:
Do you think we should add a source-constrains.html file too? Or maybe it would be better to change the title and header to indicate that it's different to the standard dynamic share page?